File: src/main/java/com/software/estudialo/entities/TestHistory.java
package com.software.estudialo.entities;


import lombok.Data;

import javax.persistence.*;
import java.sql.Timestamp;

@Entity
@Data
@Table(name = "test_history")
public class TestHistory {

    private static final long serialVersionUID = 1L;

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", unique = true, nullable = false)
    private Long id;

    @Column(name = "id_fase_item")
    private Integer idFaseItem;

    @Column(name = "id_user")
    private Integer idUser;

    @Column(name = "created_at")
    private Timestamp created_at;
}

File: src/main/webapp/src/app/models/test-history.ts
export interface ITestHistory {
  id?: number;
  idFaseItem?: number;
  idUser?: number;
  created_at?: number;
}

export class TestHistory implements ITestHistory {

  constructor(
    public id?: number,
    public idFaseItem?: number,
    public idUser?: number,
    public created_at?: number,
  ) {}
}